design/FY2019/NFP-AFNFanModel.md

```diff
@@ -0,0 +1,124 @@
+
+## E-mail and  Conference Call Conclusions ##
+
+NA
+
+## Overview ##
+
+The Fan:SystemModel object models fans of various types using a relatively simple engineering model. This fan can be used in variable air volume, constant volume, on-off cycling, two-speed, or multi-speed applications. It was designed as a replacement for Fan:ConstantVolume, Fan:OnOff, Fan:VariableVolume, and FanPerformance:NightVentilation. In order to make the airflow network model accept the Fan:SystemModel object, an additional choice of fan type input and internal code revision are needed as follows.
+
+- Additional input choice
+
+A new choice key will be added in the Supply Fan Object Type field of the AirflowNetwork:Distribution:Component:Fan object as Fan:SystemModel.
+ 
+- Code revision
+
+Two modules will be revised to accommodate the new feature: AirflowNetworkBalanceManager and HVACFan.  
+ 
+
+## Approach ##
+
+The following revisions will be performed.  Any new additions will be highlighted in red.
+
+### AirflowNetworkBalanceManager
+
+GetAirflowNetworkInput
+
+The function will be modified to accept a new choice of fan type as Fan:SystemModel. Then the code will check the input choice of the Speed Control Method field in the Fan:SystemModel object. If the input is Continuous, the AFN model will consider this fan is a VAV fan. If the input is Discrete, the AFN model will consider this fan is on/off fan. the difference between on/off and constant volume fan is based the value of LoopOnOffFanPartLoadRatio.   
+
+### HAVCFan
+
+#### Assing fan part load value
+The value of OnOffFanPartLoadFraction will be assigned in the update function.
+
+#### Create two local functions to get node numbers of inlet and outlet
+Two local functions of GetFanInletNodeNum and GetFanOutletNodeNum will be created to output node numbers. The functions will be called from GetAirflowNetworkInput.
+ 
+## Testing/Validation/Data Sources ##
+
+Test will be performed to ensure the additional fan type will work properly.
+
+## Input Output Reference Documentation ##
+
+Any new additions will be highlighted in red.
+
+\subsection{AirflowNetwork:Distribution:Component:Fan}\label{airflownetworkdistributioncomponentfan}
+
+This component represents a constant volume fan in the air distribution system (AirLoopHVAC). The air flow rate and air conditions (temperature and humidity) are obtained from the associated Fan:ConstantVolume, Fan:OnOff, or Fan:VariableVolume object.
+
+\subsubsection{Inputs}\label{inputs-16-001}
+
+\paragraph{Field: Fan Name}\label{field-fan-name-000}
+
+The name identifying an AirflowNetwork constant volume fan in an air distribution system. This name must be the same as the name of the associated Fan:ConstantVolume,~ Fan:OnOff or Fan:VariableVolume object. This name will be referenced by an AirflowNetwork:Distribution:Linkage object to represent a component.
```

Comment:
This sentence should probably be edited to either include Fan:SystemModel or to remove the types all together (since that comes up in the next field).

Reply:
@jasondegraw I am going to add Fan:SystemModel. Thanks.